CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   are included in their Conjur image.
   [cyberark/conjur#1974](https://github.com/cyberark/conjur/issues/1974)
 
+## [1.11.2] - 2020-12-17
+### Added
+- Usage of debify 2.
```

Comment:
This doesn't speak at all to the user impact of this change - if I am a user, why do I care?

I would guess in this case there is no OSS user impact? Please see our full [changelog guide](https://github.com/cyberark/community/blob/master/Conjur/CONTRIBUTING.md#changelog-guidelines) for more info on what goes into a good changelog message

Reply:
So in this case i don't need to change version at all and write anything to changelog?

Reply:
Currently Decreased version and removed info from changelog because the user expiriance with Conjur is not being changed. It is for deploy on RHEL porpuses.